### ISSUE TITLE
Surface Pro 4: Can't get touchscreen to work

### ISSUE THREAD

**Issue by atticusmas** [jakeday/linux-surface#234]
Jakeday: You've done fantastic work here, I know you have no obligation to help us folks but you truly are my hero 

The basic story is that I'm on Antergos on my SP4, and I've tried installing everything from the vanilla kernel, to pharra's fork, to dmhacker's version. I've tried manually installing the touchscreen drivers, both using the one here, and I've tried installing it from the official ipts thing here: https://github.com/ipts-linux-org/ipts-linux-new/wiki

The files I'm installing by the way are the skl version, and v78 ipts, as per the install script.

Basically, nothing seems to matter. The touchscreen simply won't respond at all. I'm rather new to this sort of debugging, so any help would be appreciated. For instance, I am not sure which logs would be useful in this situation. 

Any assistance would be great here, I really appreciate it.

**Comment by atticusmas** [comment]
Update: I installed input-utils (an AUR package), that includes lsinput. I also have tried dmesg after various attempts at triggering an input event. 

Both of these commands tell me that the touchscreen is not being recognized at all: I see nothing resembling a touch screen event with lsinput, nor do I see "Surface Pen" or the like with dmesg

**Comment by atticusmas** [comment]
Update: I am an idiot. It turns out, whatever I was doing, didn't install the kernel by default or something. 

I see an option in grub "Start linux with linux-surface4 kernel" and it's working. I have reported my own user error here, so I do apologize profusely. Thank you jakeday et al for your hard work on this fantastic kernel.
